README.md

```diff
@@ -38,13 +47,29 @@ Notifier.error('Whoops!')
 
 ### Masks
 
-``` ruby
+```ruby
 :none     # allow user interactions, don't dim background UI (default)
 :clear    # disable user interactions, don't dim background UI
 :black    # disable user interactions, dim background UI with 50% translucent black
 :gradient # disable user interactions, dim background UI with translucent radial gradient (a-la-alertView)
 ```
 
+### Setting global styles
+
+You can set the defaults for all HUDs that are created. Just set what you want to modify and it'll be applied to all HUDs displayed.
+
+```ruby
+Notifier = Motion::Blitz
+
+Notifier.background_color = UIColor.redColor
+Notifier.foreground_color = UIColor.blueColor
+Notifier.font = UIFont.fontWithName("Courier", size: 10.0)
+Notifier.info_image = 'my_info_image' # or UIImage.imageNamed('my_info_image')
+Notifier.success_image = 'my_success_image' # or UIImage.imageNamed('my_success_image')
+Notifier.error_image = 'my_error_image' # or UIImage.imageNamed('my_error_image')
+Notifier.default_mask_type = :gradient # or SVProgressHUDMaskTypeGradient
```

Comment:
Think we can use a initializer-style configuration block for these values?

Something like this perhaps?
```ruby
Motion::Blitz.configure do |config|
  config.background_color  = UIColor.redColor
  config.foreground_color  = UIColor.blueColor
  config.font              = UIFont.fontWithName('Courier', size: 10.0)
  config.info_image        = UIImage.imageNamed('my_info_image')
  config.success_image     = UIImage.imageNamed('my_success_image')
  config.error_image       = UIImage.imageNamed('my_error_image')
  config.mask_type         = :gradient
end
```

I don't think we need the `String` -> `UIImage` shortcut here. Makes sense for `Motion::Blitz.image` but these methods for global configuration are likely to be called only once within an application.